l10n/en/brands.ftl

```diff
@@ -21,6 +21,9 @@
 -brand-name-mullvad = Mullvad
 -brand-name-netscape = Netscape
 -brand-name-twitter = Twitter
+
+# X is the new rebrand name for Twitter
+-brand-name-x = X
```

Comment:
I believe we should not be using third party brand names going forward? (request from L10n team)